datastore_mongo.go

```diff
@@ -85,6 +85,23 @@ func (db *DataStoreMongo) GetDevice(id DeviceID) (*Device, error) {
 	return &res, nil
 }
 
+func (db *DataStoreMongo) DeleteDevice(id DeviceID) error {
+	s := db.session.Copy()
+	defer s.Close()
+
+	filter := bson.M{"id": id}
```

Comment:
If we change this now from `id` to `_id` this means we have to write migration on existing installations or support both at the same time. Not sure if worth the effort.